transaction-dos/src/main.rs

```diff
@@ -174,7 +175,8 @@ fn run_transactions_dos(
 
     let program_account = client.get_account(&program_id);
 
-    let message = Message::new(
+    let mut blockhash = client.get_latest_blockhash().expect("blockhash");
```

Comment:
Nit: do we need this to be mutable and reassign it every time?

Reply:
As far as I understand we need to refresh it with some periodicity (L272) to avoid filtering by time in banking.

Reply:
The same blockhash is used later to create a transaction, so I need somehow to store the value obtained earlier to pass there.